a11y-meta-display-guide/2.0/draft/techniques/epub-metadata/index.html

```diff
@@ -219,122 +219,187 @@ <h3>Check for Node</h3>
+                        </dd>
+
+                        <dt><var>audio_content</var></dt>
+                        <dd>
+                            <p>If true it indicates that the <i>accessMode="auditory"</i> is present in the package document, otherwise, if false, it means that the metadata is not present.</p>
+
+                            <p>This indicates that prerecorded audio content is included as part of the work.</p>
+                        </dd>
+
+
+                    </dl>
+                    <h5>Variables setup</h5>
+                    <ol class="condition">
+                        <li><b>LET</b> <var>package_document</var> be the result of calling <a href="#pre-processing">preprocessing</a> given <var>package_document_as_text</var>.</li>
+
+                        <li><b>LET</b> <var>all_content_audio</var> be the result of calling <a href="#check-for-node">check for node</a> on <var>package_document</var>, <code class="xpath">/package/metadata/meta[@property="schema:<i>accessModeSufficient</i>" and normalize-space()="<i>auditory</i>"]</code>.</li>
+
+                        <li><b>LET</b> <var>synchronised_pre_recorded_audio</var> be the result of calling <a href="#check-for-node">check for node</a> on <var>package_document</var>, <code class="xpath">/package/metadata/meta[@property="schema:<i>accessibilityFeature</i>" and normalize-space()="<i>sychronizedAudioText</i>"]</code>.</li>
+
+                        <li><b>LET</b> <var>audio_content</var> be the result of calling <a href="#check-for-node">check for node</a> on <var>package_document</var>, <code class="xpath">/package/metadata/meta[@property="schema:<i>accessMode</i>" and normalize-space()="<i>auditory</i>"]</code>.</li>
+
+                    </ol>
+                    <h5>Instructions</h5>
+                    <ol class="condition">
+                        <li>
+                            <span><b>IF</b> <var>all_content_audio</var>:</span>
+                            <span><b>THEN</b> display <code id="ways-of-reading-prerecorded-audio-only">"Audio only"</code>.</span>
+                        </li>
+                        <li>
+                            <span><b>ELSE IF</b> <var>synchronised_pre_recorded_audio</var>:</span>
+                            <span><b>THEN</b> display <code id="ways-of-reading-prerecorded-audio-synchronized">"Prerecorded audio synchronized with text"</code>.</span>
+                        </li>
+                        <li>
+                            <span><b>ELSE IF</b> <var>audio_content</var>:</span>
+                            <span><b>THEN</b> display <code id="ways-of-reading-prerecorded-audio-complementary">"Complementary audio and text"</code>.</span>
+                        </li>
 
+                        <li>
+                            <b>ELSE</b> either omit this key information if metadata is missing or display <code id="ways-of-reading-prerecorded-audio-no-metadata">"No information is available"</code>.
```

Comment:
Same issue with outputting the generic "no information is available" string here.